mods/disable-ms-teams-hotkey.wh.cpp

```diff
@@ -0,0 +1,48 @@
+// ==WindhawkMod==
+// @id              disable-ms-teams-hotkey
+// @name            Disable Microsoft Teams hotkeys used by Visual Studio
+// @description     Disables Ctrl+Shift+Space that does something useful in Visual Studio and not in Teams.
+// @version         1.0.0
+// @author          Yves Goergen
+// @github          https://github.com/ygoe
+// @homepage        https://ygoe.de/
+// @include         ms-teams.exe
+// ==/WindhawkMod==
+
+// ==WindhawkModReadme==
+/*
+# Disable Microsoft Teams hotkeys used by Visual Studio
+
+Microsoft Teams occupies the keyboard shortcut Ctrl+Shift+Space to do whatever nonsense with it while sharing the screen.
+This hotkey is also assigned in Visual Studio by default to show the parameters info of the method under the cursor.
+That information is often very useful while coding, but it's not available in a team coding session when sharing the screen.
+So this mod prevents Teams from ever occupying that hotkey so that it remains available to Visual Studio like it's supposed to.
+*/
+// ==/WindhawkModReadme==
+
+BOOL(*pOriginalRegisterHotKey)(HWND hWnd, int id, UINT fsModifiers, UINT vk);
```

Comment:
Here too:
```suggestion
BOOL(WINAPI *pOriginalRegisterHotKey)(HWND hWnd, int id, UINT fsModifiers, UINT vk);
```

Reply:
Alright, I don't know enough of the details of C. But I can't accept your suggestion because the GitHub UI doesn't show me the button in a too small container. Please accept it without my formal consent.

Reply:
Yes, I have this problem too, looks like a GitHub bug. Just update the source code by adding the missing keyword, thanks.

Reply:
Your reply comment gave the box enough space to fit the accept button. 🙂